Sources/Styleguide/RegisterFonts.swift

```diff
@@ -34,7 +34,7 @@ extension UIFont {
         \(error.map { " \($0.takeUnretainedValue().localizedDescription)" } ?? "")
         """
       )
-      return false
+      return true
```

Comment:
This branch should still return true, otherwise the `XCTSkip` logic hits after the first snapshot test.